test/test_nn.py

```diff
@@ -1834,6 +1834,13 @@ def check():
         parameters.pop('p4')
         check()
 
+        p = Parameter(torch.randn(10, 10))
+        parameter_dict['p12'] = p
+        p_popitem = parameter_dict.popitem()
+        self.assertIs(p_popitem[0], 'p12')
```

Comment:
shouldn't this be assertEquals? assertIs checks that they are the same object reference

Reply:
Since strings are immutable objects, I don't think there is a difference between value vs. object comparison.

But I do agree assertEquals is better in terms of code style. Fixed now!